src/main/java/org/jnode/fs/xfs/directory/NodeDirectory.java

```diff
@@ -67,20 +71,15 @@ public NodeDirectory(byte[] data, long offset, XfsFileSystem fileSystem, long iN
         this.leafExtentIndex = (int) leafExtentIndex;
     }
 
-    public long getiNodeNumber() {
-        return iNodeNumber;
-    }
-
-    public int getLeafExtentIndex() {
-        return leafExtentIndex;
-    }
-
     /**
      * Get the node block entries
      *
      * @return a list of inode entries
      */
     public List<FSEntry> getEntries(FSDirectory parentDirectory) throws IOException {
+        // TODO,
+        //  1. what does the 120 mean here? Average entry count per extent?
```

Comment:
Ugh, no "120" in the docs either. I'd hate to try to follow this logic and code, probably something we won't be able to get done this time around (unless you're really inspired to, that'd be great 😅 )

Reply:
Actually, the only use of this number is give an init size to the array (to prevent copying too much when the array grows? I don't know). We can even remove it if we don't care about the performance. But I'll leave it for now.